src/components/Charts/ChartVaccinations.tsx

```diff
@@ -1,37 +1,245 @@
-import React from 'react';
-import ChartZones from './ChartZones';
-import { VACCINATIONS_LEVEL_INFO_MAP } from 'common/metrics/vaccinations';
+import React, { useCallback, Fragment } from 'react';
+import moment from 'moment';
+import { isNumber } from 'lodash';
+import { AxisLeft, AxisBottom } from '@vx/axis';
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleUtc, scaleLinear } from '@vx/scale';
+import { useTooltip } from '@vx/tooltip';
+import { formatPercent, formatUtcDate } from 'common/utils';
 import { Column } from 'common/models/Projection';
-import { formatPercent } from 'common/utils';
-import { getMetricDefinition, Metric } from 'common/metric';
+import { Tooltip, RectClipGroup } from 'components/Charts';
+import { ScreenshotReady } from 'components/Screenshot';
+import GridLines from 'components/Explore/GridLines';
+import * as Styles from 'components/Explore/Explore.style';
+import { Series } from 'components/Explore/interfaces';
+import ChartSeries, { SeriesMarker } from 'components/Explore/SeriesChart';
+import ChartOverlay from 'components/Explore/ChartOverlay';
+import { findPointByDate, getTimeAxisTicks } from 'components/Explore/utils';
+import * as ChartStyle from './Charts.style';
 
-const CAP_Y = 1;
+const getDate = (d: Column) => new Date(d.x);
+const getY = (d: Column) => d.y;
 
-const getPointText = (valueY: number) => formatPercent(valueY, 1);
+const xTickFormat = (date: Date, isMobile: boolean) => {
+  const momentDate = moment(date);
+  // Shows the year if the tick is in January (0) or December (11)
+  const dateFormat =
+    momentDate.month() === 0 || momentDate.month() === 11 ? 'MMM YYYY' : 'MMM';
+  return momentDate.format(dateFormat);
+};
+
+const VaccinesTooltip: React.FC<{
+  date: Date;
+  seriesList: Series[];
+  left: (d: Column) => number;
```

Comment:
fwiw I didn't know what left and top were here until i read the calling code - maybe `marginLeft` and `marginTop`?

Reply:
Not really, because those are `top` and `left` to absolutely position the tooltip